app/translations/en.json

```
@@ -22,6 +25,37 @@
         "servicesStatus": "Services Status",
         "systemStatus": "System Status"
     },
+    "gettingStartedModal" : {
+        "modal": {
+            "disableModalLabel": "Don't show next time",
+            "enableModalHelp": "You can enable modal always in user profile.",
+            "closeModal": "Close",
+            "stepBack": "Back",
+            "stepNext": "Next",
+            "stepFinish": "Finish"
+        },
+        "initialization": {
+            "pluginsLoadingError": "Plugins information loading error.",
+            "secretsLoadingError": "Secrets information loading error."
+        },
+        "validation": {
+            "someTechnologyRequiredError": "Please select some technology.",
+            "allSecretsRequiredError": "Please type all secrets."
+        },
+        "summary": {
+            "taskListTitle": "Task List",
+            "pluginInstallationMessageSuffix": "plugin will be installed.",
+            "secretCreateMessageSuffix": "secret will be created",
+            "secretUpdateMessageSuffix": "secret will be updated"
```

I think they all should use the same style,
either without dots:
```suggestion
            "pluginInstallationMessageSuffix": "plugin will be installed",
            "secretCreateMessageSuffix": "secret will be created",
            "secretUpdateMessageSuffix": "secret will be updated"
```
or with dots: 
```suggestion
            "pluginInstallationMessageSuffix": "plugin will be installed.",
            "secretCreateMessageSuffix": "secret will be created.",
            "secretUpdateMessageSuffix": "secret will be updated."
```

---

fixed